pkg/tests/tasks/security/authorization/ext_auth_test.go

```diff
@@ -136,7 +136,25 @@ func TestEnvoyExtAuthzGrpcExtensionProvider(t *testing.T) {
 		oc.WaitDeploymentRolloutComplete(t, ns, "ext-authz")
 
 		t.LogStep("Set envoyExtAuthzgRPC extension provider in SMCP")
-		if env.GetSMCPVersion().GreaterThanOrEqual(version.SMCP_2_3) {
+		if env.GetSMCPVersion().GreaterThanOrEqual(version.SMCP_2_4) {
```

Comment:
```suggestion
		if env.GetSMCPVersion().LessThan(version.SMCP_2_4) {
```
This should be `LessThan` and not `GreaterThanOrEqual`, as it is on line 57.